pandora-server-directory/src/database/mongoDb.ts

```diff
@@ -518,6 +518,22 @@ export default class MongoDatabase implements PandoraDatabase {
 
 	private async _doMigrations(): Promise<void> {
 		// insert migration code here
+		for await (const account of this._accounts.find().stream()) {
+			if (account.directMessages == null)
+				continue;
+
+			const directMessages: typeof account.directMessages = [];
+			for (const dm of account.directMessages) {
+				const migrated = {
```

Comment:
Sooo... here you create `migrated`... and then?

Reply:
...and then it disappears...